File: tools_java/tools/src/tools/Cleanup.java
package tools;
/**
 * 
 * @author broihier
 * <pre>
 * Class for creating Cleanup objects
 * </pre>
 */
public class Cleanup extends DatabaseComponent {
	/** 
	 * Constructor for creating Cleanup objects
	 * 
	 * @param category name that maps to 'category'.testDbPost files
	 * 
	 */
	public Cleanup(String category) {
		super(category + ".testDbPost");
	}
	/**
	 * Method to get the cleanup text for a particular testID
	 * @param testID test case ID
	 * @return getComponentText(testID)
	 */
	public String getCleanupText(String testID) {
		return getComponentText(testID);
	}

}

File: tools_C++/src/Cleanup.cpp
/*
 * Cleanup.cpp
 *
 *  Created on: Apr 12, 2017
 *      Author: broihier
 */

#include "Cleanup.h"

Cleanup::Cleanup(std::string category): DatabaseComponent(category+".testDbPost") {
}

Cleanup::~Cleanup() {
}

const std::string& Cleanup::getCleanupText(const std::string& id) const {
	return getComponentText(id);
}
